commands/echo/index.js

```diff
@@ -1,3 +1,8 @@
-module.exports = function (opts) {
-  opts.event.channel.send(opts.argsRaw.join(' '));
-}
+module.exports = {
+  description: 'Reply with the arguments',
+  usage: 'string...',
+  aliases: ['print', 'say'],
+	execute(opts) {
+    opts.event.channel.send(opts.argsRaw.join(' '));
+	},
+};
```

Comment:
On this file, you can see the change between the v1 and v2 of the command structure